packages/material-ui/src/NativeSelect/index.d.ts

```diff
@@ -1,2 +1,4 @@
 export { default } from './NativeSelect';
 export * from './NativeSelect';
+export { default as nativeSelectClasses } from './nativeSelectClasses';
```

Comment:
Convention with the other files

```suggestion

export { default as nativeSelectClasses } from './nativeSelectClasses';
````